Cour-de-cassation/label
# adding decisionMetadataGenerator for test

## Issue description :
Adding an new decisonMetadataGenerator for rewrite tests document

## Describe your changes :

- Create an decisionMetadataGenerator in documentGenerator.ts file
- Exporting the function

## How to test :

- I write an example in cleanDuplicatedDocuments.spec.ts test file, go to this file and check the implementation of decisionmetadata in documentModule.generator.generate function
- cd packages/generic/backend/src/app/scripts/cleanDocuments
- Run yarn test cleanDuplicatedDocuments.spec.ts

## Checklist before requesting a review
